src/ci/bin/testCentaurAws.sh

```diff
@@ -34,6 +34,8 @@ cromwell::build::run_centaur \
     -e abort.scheduled_abort \
     -e relative_output_paths \
     -e relative_output_paths_colliding \
-    -e standard_output_paths_colliding_prevented
+    -e standard_output_paths_colliding_prevented \
+    -e space \
```

Comment:
Do we want to put a link to the ticket here (so that we can know when it's safe to remove these exclusions?)